config/common/council_tasks.cwt

```diff
@@ -0,0 +1,165 @@
+types = {
+    type[council_task] = {
+        path = "game/common/council_tasks"
+    }
+}
+
+council_task = {
+	## cardinality = 0..1
+	### Is the defualt task for the specified position (no by default, only applicable on task_type_general with task_progress_infinite)
+	default_task = bool
+	### Position that can perform this task (positions can be found in common/council_positions)
+	position = <council_position> 
+	### Type of the task [task_type_general|task_type_county|task_type_court]
+	task_type = enum[task_types]
+	## cardinality = 0..1
+	### What can a county task target? Not relevant for other task-types. Valid: all, realm, domain, neighbor_land, neighbor_land_or_water. neighbor_land_or_water only works for the AI
```

Comment:
I just copied PDX's documentation and made minor edits. In CK2, the aliases of ai_county_target was applied on top of the aliases of county_target, so it would be logical to put neighbor_land_or_water in both of them but it only works for the AI on water, while it might be different for CK3.

Reply:
Ohh I mis-read the doc string. I thought it was saying that the whole key only affected the ai. I see now that it's saying specifically only neighbor land or water is ai only.